docs/en/user_guides/yolov5_tutorial.md

```diff
@@ -1 +1,249 @@
-# From Getting Started to Deployment tutorial with YOLOv5
+# YOLOv5: From Start to Deploy
+
+## Environment Setup
+
+Note: Since this repository uses OpenMMLab 2.0, please create a new conda virtual environment to prevent conflicts with your existing repositories and projects of OpenMMLab 1.0.
+
+```shell
+conda create -n open-mmlab python=3.8 -y
+conda activate open-mmlab
+conda install pytorch torchvision -c pytorch
+# conda install pytorch torchvision cpuonly -c pytorch
+pip install -U openmim
+mim install mmengine
+mim install "mmcv>=2.0.0rc1,<2.1.0"
+mim install "mmdet>=3.0.0rc0,<3.1.0"
+# for albumentations
+git clone https://github.com/open-mmlab/mmyolo.git
+cd mmyolo
+# Install albumentations
+pip install -r requirements/albu.txt
+# Install MMYOLO
+mim install -v -e .
+# "-v" means verbose, or more output
+# "-e" means install the project in editable mode, so any local modifications made to the code will take effect, eliminating the need to reinstall.
+```
+
+For more detailed information about environment configuration, please refer to [get_started](../get_started.md).
+
+## Dataset Preparation
+
+In this tutorial, the training dataset for MMYOLO is less than 40MB and is selected from the balloon dataset.
```

Comment:
In this tutorial, we provide the ballon dataset, which is less than 40MB, as the training dataset for MMYOLO.